app/src/controllers/organizations/projects/selectors.js

```diff
@@ -38,8 +38,8 @@ export const createOrganizationProjectsParametersSelector = ({
       defaultSorting,
       sortingKey,
     }),
-    ({ [SIZE_KEY]: limit, [SORTING_KEY]: sort, [PAGE_KEY]: pageNumber }) => {
-      return getAlternativePaginationAndSortParams(sort, limit, pageNumber);
+    ({ [SIZE_KEY]: limit, [SORTING_KEY]: sort, [PAGE_KEY]: pageNumber, [SEARCH_KEY]: name }) => {
+      return getAlternativePaginationAndSortParams(sort, limit, pageNumber, name);
```

Comment:
Let's not mutate the `getAlternativePaginationAndSortParams` to support filter fields. This function is intended just to convert the new pagination and sort format to unify the approach with old APIs. So if you need to propagate all the rest of query parameters without changes, just return them as a part of the result object, e.g.
```suggestion
    ({ [SIZE_KEY]: limit, [SORTING_KEY]: sort, [PAGE_KEY]: pageNumber, ...rest }) => {
      return { ...getAlternativePaginationAndSortParams(sort, limit, pageNumber), ...rest };
```